.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
nit: Add to .gitignore?

Reply:
Just removed the file for now. I added this to gitignore in a future PR